### PR TITLE
Add a bake task for creating policies

### PR DESCRIPTION
**Pull request by markstory**
If the user has bake and authorization installed we can provide them with a task to generate policy classes. I've included stub methods for entities as I think they will be the most common type of policy created.

### CODE REVIEW

File: src/Template/Bake/Element/entity_methods.twig [hunk 1]
```diff
@@ -0,0 +1,43 @@
+    /**
+     * Check if $user can create {{ name }}
+     *
+     * @param Authorization\IdentityInterface $user The user.
+     * @param {{ namespace }}\Model\Entity\{{ name }} ${{ variable_name }}
+     * @return bool
+     */
+    public function canCreate(IdentityInterface $user, ${{ variable_name }})
```
Comment (robertpustulka) [inline src/Template/Bake/Element/entity_methods.twig]: Should the entity be type hinted as entity class name or `EntityInterface`?

Comment (markstory) [reply src/Template/Bake/Element/entity_methods.twig]: I can add typehints for the entity classes.

**Comment by markstory** [comment]
@robertpustulka @burzum Is this good to go?

**Comment by robertpustulka** [comment]
Looks OK

**Pull request merged**